packages/lit-dev-server/src/middleware/tutorials-middleware.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import type Koa from 'koa';
+
+/**
+ * Koa Middleware to serve '/tutorials/view.html' from '/tutorials/*'.
+ *
+ * @param staticRoot Root of the static server
+ */
+export const tutorialsMiddleware = (): Koa.Middleware => async (ctx, next) => {
+  const path = ctx.request.path;
+
+  /**
+   * We want to intercept /tutorials/* but not /tutorials/ itself or the place
+   * where we are currently rendering the markdown to html so not
+   * /tutorials/content/*
+   */
+  if (
+    path.startsWith('/tutorials/') &&
+    !path.startsWith('/tutorials/content/') &&
+    path !== '/tutorials/'
+  ) {
+    // Serve /tutorials/view/
```

Comment:
Might be a slightly unclear comment, since that's neither the URL nor the file. Probably don't need a comment IMO.